Update SSH for Access for Infrastructure dev docs (#17415)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/connections/connect-networks/use-cases/ssh/ssh-infrastructure-access.mdx

To connect your devices to Cloudflare:

1. [Deploy the WARP client](/cloudflare-one/connections/connect-devices/warp/deployment/) on your devices in Gateway with WARP mode.

> 2. [Enable proxy mode for WARP](/cloudflare-one/connections/connect-networks/warp/warp-modes/#proxy-mode).

Reviewer: @asamborski I'm confused by this step because WARP cannot simultaneously be in Gateway with WARP mode and Proxy mode. Are you referring to the Gateway proxy for TCP? https://developers.cloudflare.com/cloudflare-one/policies/gateway/proxy/#enable-the-gateway-proxy

Author: I'm referring to Settings -> Network -> Firewall -> Proxy. It looks like we may be discussing the same thing?

Reviewer: Yes, we're talking about the same thing. The link in the docs is for something different.
Does Infrastructure Access only require the TCP proxy, or also UDP and ICMP?

Author: TCP only!

> 2. [Enable proxy mode for WARP](/cloudflare-one/connections/connect-networks/warp/warp-modes/#proxy-mode).

Reviewer: ```suggestion
2. [Enable the Gateway proxy for TCP](/cloudflare-one/policies/gateway/proxy/#enable-the-gateway-proxy).
```

> 3. Install and trust the [Cloudflare root certificate](/cloudflare-one/connections/connect-devices/warp/user-side-certificates/) on your devices.

Reviewer: ```suggestion
```

> 4. [Create device enrollment rules](/cloudflare-one/connections/connect-devices/warp/deployment/device-enrollment/) to determine which devices can enroll to your Zero Trust organization.

Reviewer: ```suggestion
3. [Create device enrollment rules](/cloudflare-one/connections/connect-devices/warp/deployment/device-enrollment/) to determine which devices can enroll to your Zero Trust organization.
```

### Modify your SSHD config
<Render file="ssh/modify-sshd" />

:::note

> For certain distributions, such as Amazon Linux 1 (based on RHEL), the certificate file permissions must be set to 600. "

Reviewer: How do you set file permissions to 600?

Author: will add that in!

Author: Added here: https://github.com/cloudflare/cloudflare-docs/pull/17415/commits/30c4bdd918dbde65a799e488d2cb7fcf61c169fe
